### DISCUSSION TITLE
Script from play with docker not work

### DISCUSSION THREAD

**Discussion by uluumbch** [caprover/caprover#2176] [Q&A]
I tried to ssh into the instance and then run `curl -L https://pwd.caprover.com | bash` but it is returning error `ERROR: this script is only meant to be used on play-with-docker.com environment` do I do it wrong or smth?

![image](https://github.com/user-attachments/assets/db4f5b0d-7081-49d7-9009-eb678565aaed)

**Comment by githubsaturn** [comment]
This script is meant for https://labs.play-with-docker.com/
